xcm/xcm-simulator/example/src/lib.rs

```diff
@@ -166,7 +166,7 @@ mod tests {
 		ParaA::execute_with(|| {
 			assert_ok!(ParachainPalletXcm::send_xcm(
 				Here,
-				MultiLocation::new(1, X1(Parachain(2))),
+				(1, Parachain(2)),
```

Comment:
This makes me feel like I should implement `From<(Parent, Junction)>` for `MultiLocation`. Can do in this PR or in a future one.

Reply:
Done in #3635.